pipeline/terraform/main.tf

```diff
@@ -1,8 +1,75 @@
+module "catalogue_pipeline_2021-03-05" {
+  source = "./stack"
+
+  pipeline_date = "2021-03-05"
+  release_label = "stage"
+
+  # Transformer config
+  #
+  # If this pipeline is meant to be reindexed, remember to uncomment the
+  # reindexer topic names.
+
+  is_reindexing = false
```

Comment:
Typically I reindex with this set to “true” – it adds more tasks and turns up RDS and the pipeline storage cluster.

might be interesting to see wa reindex with rate limiting, but it might be very slow

Reply:
I have done that locally, wasn't going to commit it but actually I guess I may as well

Reply:
Ah, doesn't need to be committed, as long as you've got it.